ledger/ledger-api-client/src/main/scala/com/digitalasset/ledger/client/services/commands/CommandSubmissionFlow.scala

```diff
@@ -40,9 +41,9 @@ object CommandSubmissionFlow {
   def getStatus(tryResponse: Try[Empty]): Try[Status] = {
     tryResponse.fold[Try[Status]](
       {
-        case GrpcStatusError(status) =>
-          Success(Status(status.getCode.value(), status.getDescription))
-        case other => Failure(other)
+        case GrpcException(GrpcStatus(code, description), _) =>
+          Success(Status(code.value(), description.getOrElse("")))
```

Comment:
This pattern is everywhere. Perhaps extract it to the library with a helper?

I don't think it's possible to overload the `unapply` to emit either an `io.grpc.Status` or a `com.google.rpc.status.Status` without some funky implicits, but that could also be an option if you know a nice way of doing it.

Reply:
Unfortunately it's not possible to overload based on the result type. I thought about adding an helper, I guess it could make sense adding it to the new library.

Reply:
Cool stuff: this method was actually not used anywhere. I deleted it. The helper is already done, so I'll check if it's actually used somewhere, otherwise I'll just revert. :wink:

Reply:
It's used in a couple of places. I'm a proponent of the YCRYOOT principle (_You Can Repeat Yourself Once Or Twice_) but I guess the helpers makes sense, so I'll leave it there.

Reply:
:thinking: I should find a better name for YCRYOOT.

Reply:
Addressed by https://github.com/digital-asset/daml/pull/3344/commits/4d55e0bfcaeee0308df54cc495c690e88693f275